runtime/v2/runc/v1/service.go

```diff
@@ -271,11 +271,11 @@ func (s *service) Start(ctx context.Context, r *taskAPI.StartRequest) (*taskAPI.
 		s.eventSendMu.Unlock()
 		return nil, errdefs.ToGRPC(err)
 	}
-	if err := s.ep.Add(container.ID, container.Cgroup()); err != nil {
-		logrus.WithError(err).Error("add cg to OOM monitor")
-	}
 	switch r.ExecID {
 	case "":
+		if err := s.ep.Add(container.ID, cg); err != nil {
```

Comment:
CI failed, it's `undeclared name: cg`

`s.ep.Add(container.ID, container.Cgroup());`